libraries/chain/include/eosio/chain/types.hpp

```diff
@@ -82,6 +81,16 @@ namespace eosio { namespace chain {
    using private_key_type = fc::crypto::private_key;
    using signature_type   = fc::crypto::signature;
 
+#if BOOST_VERSION >= 107100
```

Comment:
Defined in boost/version.hpp.

Reply:
Added include and include of `<boost/container/deque.hpp>`